.github/workflows/deploy.sh

```diff
@@ -60,7 +60,7 @@ else
 fi
 
 #######################################
-# UKEF deployment shell script v0.0.6
-# 11/05/2023
+# UKEF deployment shell script v0.0.7
+# 31/05/2023
```

Comment:
02/06/2023?

Reply:
Although looks like this version/script is not specific to EXIP so makes sense to keep the date the same as other usages